google/cloud/aiplatform/training_jobs.py

```diff
@@ -3346,3 +3347,255 @@ def _run(
         )
 
         return model
+
+
+class AutoMLTextTrainingJob(_TrainingJob):
+    def __init__(
+        self,
+        display_name: str,
+        prediction_type: str,
+        multi_label: Optional[bool] = None,
+        sentiment_max: Optional[str] = None,
+        model_hint: str = "default",
```

Comment:
This seems to be missing in the docstring.

Reply:
Removed